src/utils.rs

```diff
@@ -0,0 +1,9 @@
+use crate::client::queries::query_me::WalletCurrency;
+use crate::types::Wallet;
+
+pub fn wallet_to_currency(wallet: &Wallet) -> WalletCurrency {
+    match wallet {
+        Wallet::Usd => WalletCurrency::USD,
+        Wallet::Btc => WalletCurrency::BTC,
+    }
+}
```

Comment:
You should use Rust From or Into traits for this.
https://doc.rust-lang.org/rust-by-example/conversion/from_into.html

Reply:
Removed this function and implemented `From` for `Wallet`
3292bb9e8fcc3418cb86e6f308298920aea41861